scripts/integration-tests/interactive-integration-env

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
This scripts is really just a helper script "around" all your work, so I would put it in `scripts/`, instead of `scripts/integration-tests` for consistency.